### ISSUE TITLE
Incorrect mapping between DynamoDB attributes and TA keys

### ISSUE THREAD

**Issue by mlnkrlc** [awslabs/aws-limit-monitor#23]
Example key from latest version of Limit Monitor:
```
{
  "AccountId": "123456789012",
  "CurrentUsage": "Green", <-
  "ExpiryTime": 1560964756524,
  "LimitAmount": "0", <-
  "LimitName": "1000000", <-
  "MessageId": "02102f5b-01cf-4f88-86b2-3cda37bd1fe5",
  "Region": "ap-south-1",
  "Service": "EBS",
  "Status": "OK",
  "TimeStamp": "2019-06-04T17:16:04Z"
}
```
Culprit seems to be [here](https://github.com/awslabs/aws-limit-monitor/blob/master/source/services/limitreport/lib/limit-report.js#L156-L158), as Slack notifications are referencing proper [attributes](https://github.com/awslabs/aws-limit-monitor/blob/master/source/services/slacknotify/lib/slack-notify.js#L133-L148)

**Comment by gsingh04** [comment]
We have added your request to our solution backlog items and it will be considered in future
solution releases. We thank you for your contribution.

**Comment by alexromero88** [comment]
Hi @gsingh04 , is there an estimation of when this fix will be released? Thanks

**Comment by gsingh04** [comment]
This has been released as part of v5.2.
